Sprint-3/revise/implement/isPrime.js

```diff
@@ -0,0 +1,16 @@
+function isPrime(num){
+    if(num <= 1){
+        return `${num} is not prime`;
+    }
+    
+    for(let i = 2; i <= Math.sqrt(num); i++ ){
```

Comment:
You can possibly improve the performance of the code in the following manners:
- Check if num is 2, and check only odd numbers >= 3 in the loop
- Avoid calling `Math.sqrt(num)` repeatedly by assigning the value of `Math.sqrt(num)` to a variable once, and then refer to the variable in the condition of the loop.
  - Note: The condition is checked at the start of every iteration.